src/aliases.php

```diff
@@ -40,26 +41,3 @@ class_alias($class, $alias);
 class Google_Task_Composer extends \Google\Task\Composer
 {
 }
-
-if (\false) {
-  class Google_AccessToken_Revoke extends \Google\AccessToken\Revoke {}
-  class Google_AccessToken_Verify extends \Google\AccessToken\Verify {}
-  class Google_AuthHandler_AuthHandlerFactory extends \Google\AuthHandler\AuthHandlerFactory {}
-  class Google_AuthHandler_Guzzle5AuthHandler extends \Google\AuthHandler\Guzzle5AuthHandler {}
-  class Google_AuthHandler_Guzzle6AuthHandler extends \Google\AuthHandler\Guzzle6AuthHandler {}
-  class Google_AuthHandler_Guzzle7AuthHandler extends \Google\AuthHandler\Guzzle7AuthHandler {}
-  class Google_Client extends \Google\Client {}
-  class Google_Collection extends \Google\Collection {}
-  class Google_Exception extends \Google\Exception {}
-  class Google_Http_Batch extends \Google\Http\Batch {}
-  class Google_Http_MediaFileUpload extends \Google\Http\MediaFileUpload {}
-  class Google_Http_REST extends \Google\Http\REST {}
-  class Google_Model extends \Google\Model {}
-  class Google_Service extends \Google\Service {}
-  class Google_Service_Exception extends \Google\Service\Exception {}
-  class Google_Service_Resource extends \Google\Service\Resource {}
-  class Google_Task_Exception extends \Google\Task\Exception {}
-  interface Google_Task_Retryable extends \Google\Task\Retryable {}
-  class Google_Task_Runner extends \Google\Task\Runner {}
-  class Google_Utils_UriTemplate extends \Google\Utils\UriTemplate {}
-}
```

Comment:
This is here for IDEs, without it they don't know the aliases exist

Reply:
@bshaffer strange solution.. why not stubs?

Reply:
@bshaffer also it's not correct because we don't have that's classes, it's must be something like `use Google\AccessToken\Revoke as Google_AccessToken_Revoke;`

Reply:
These classes are added as aliases in the code above this. That's why `aliases.php` exists, to create the aliases that are defined in this list. See #1995

Reply:
@bshaffer it's not same:
~~~php
class A {}
class_alias(A::class, 'B');
var_dump(get_class(new B()) === B::class); // false
class C extends A {}
var_dump(get_class(new C()) === C::class); // true
~~~

Reply:
@bshaffer we can generate stubs for IDE, see https://psalm.dev/docs/running_psalm/plugins/authoring_plugins/

Reply:
I do not understand what you're trying to achieve or what you think the problem is with the current implementation. I'd prefer to keep it as is unless there is a specific problem you're trying to fix.

Reply:
@bshaffer I'm removed redundant code, for ide need generate stubs

Reply:
These lines are not redundant. They are here so IDEs know the aliases exist.

I don't know what generating stubs would accomplish that is not already accomplished here by the lines of code you removed.